db/seeds.rb

```diff
@@ -6,8 +6,16 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-Lifehack.create(
-  [{ title: 'How to tie a tie',
-    description: 'tie a knot' },
-    { title: 'How to tie a tie1' }]
-)
+User.create([{
+  first_name: 'Jon',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.